src/locale.js

```diff
@@ -123,19 +167,45 @@ export default class Locale {
 
 	/**
 	 * Base for the {@link #t} method.
-	 *
 	 * @private
 	 */
-	_t( str, values ) {
-		let translatedString = translate( this.uiLanguage, str );
+	_t( msgid = '', values = [] ) {
+		return translate( this.uiLanguage, msgid )
```

Comment:
Why not have the same functionallity here and in `_ct` and `_ctn` (splitting to array?).

We could move lines 183-200 to a seprate function and use them in `_t`, `_ct` and `_ctn`.